tests/automation_framework/tests/work_order_tests/test_submit.py

```diff
@@ -529,19 +499,18 @@ def test_work_order_signing_wrong(self):
 
         result_response = self.test_obj.getresult(
```

Comment:
For the work_order_signing_wrong, the test case was failing and now it's passing? I see that you have changed the assert from check_negative_response to verify_test

Reply:
Reverting back to check_negative_test as i have to revisit the test to get confirmation from Ram on the behavior.

Reply:
changes addressed in commit "3010f9e43d2d6a3cea2f19a464bd6efbae48e964"